.travis.yml

```diff
@@ -1,36 +1,55 @@
 language: python
 
-sudo: false
-
-python:
-  - 3.5
-
-env:
-  - export FAIL_ON_EXTERNAL_DEPRECATION='False'
-  - export FAIL_ON_EXTERNAL_DEPRECATION='True'
-
 matrix:
+  include:
+  - python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='False'
+  - python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='True'
+  - os: osx
+    language: generic
+    python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='False'
+  - os: osx
+    language: generic
+    python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='True'
   allow_failures:
-    - env: export FAIL_ON_EXTERNAL_DEPRECATION='True'
+  - env: export FAIL_ON_EXTERNAL_DEPRECATION='True'
 
 before_install:
-  # Does not matter which miniconda we get, we still create a virtualenv
-  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-  - chmod +x miniconda.sh
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - hash -r
-  # miniconda is not always up-to-date with conda.
-  - conda update --yes conda
+- if [[ $TRAVIS_OS_NAME == 'osx' ]] ; then curl "https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh" -o miniconda.sh; else wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh; fi
+- chmod +x miniconda.sh
+- "./miniconda.sh -b -p $HOME/miniconda"
+- export PATH="$HOME/miniconda/bin:$PATH"
+- hash -r
+- conda update --yes conda
 
 install:
-  - DEPS="nose numpy scipy matplotlib ipython h5py sympy scikit-learn dill natsort setuptools scikit-image"
-  - conda create -n testenv --yes python=$TRAVIS_PYTHON_VERSION $DEPS
```

Comment:
`python=$TRAVIS_PYTHON_VERSION` does not add anything? and is not understood at osx. So it had to go